### PR TITLE
Change Koji URLs used in staging

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

**Review by mfocko — APPROVED** [review]

**Comment by lbarcziova** [comment]
recheck

**Review by jpopelka — APPROVED** [review]

**Comment by csomh** [comment]
Why not make this become part of packit-service configs?

**Comment by lbarcziova** [comment]
Do you mean to have `koji_build_logs_url` and `koji_build_web_url` in the config?

**Comment by csomh** [comment]
> Do you mean to have `koji_build_logs_url` and `koji_build_web_url` in the config?

Yes, exactly. Though I'm not sure how much that would complicate things.

File: packit_service/worker/build/koji_build.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def run_build(
             task_id = int(task_id_match.group(1))
 
         task_url_match = search(
-            pattern=r"(https://koji\.fedoraproject\.org/koji/taskinfo\?taskID=\d+)",
+            pattern=r"(https://koji\.(stg\.)?fedoraproject\.org/koji/taskinfo\?taskID=\d+)",
```
Comment (csomh) [inline packit_service/worker/build/koji_build.py:226]: How about using something like below?

```suggestion
            pattern=r"(https://.+/koji/taskinfo\?taskID=\d+)",
```

The above would work with any Koji instance which has a `/koji/taskinfo?taskID=` endpoint.

**Review by csomh — APPROVED** [review]
One more recommendation for improvement, otherwise looks awesome :smiley: 

Thank you! :rocket:

**Pull request merged**